app/src/main/java/com/android/voyageur/ui/trip/settings/Settings.kt

```diff
@@ -0,0 +1,57 @@
+package com.android.voyageur.ui.trip.settings
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Home
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import com.android.voyageur.model.trip.TripsViewModel
+import com.android.voyageur.ui.navigation.BottomNavigationMenu
+import com.android.voyageur.ui.navigation.LIST_TRIP_LEVEL_DESTINATION
+import com.android.voyageur.ui.navigation.NavigationActions
+import com.android.voyageur.ui.navigation.Screen
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SettingsScreen(
+    tripsViewModel: TripsViewModel,
+    navigationActions: NavigationActions,
+) {
+  val trip by tripsViewModel.selectedTrip.collectAsState()
+
+  Scaffold(
+      modifier = Modifier.testTag("settingsScreen"),
+      topBar = {
+        TopAppBar(
+            modifier = Modifier.testTag("topBar"),
+            title = { Text("Settings:") },
+            navigationIcon = {
+              IconButton(
+                  modifier = Modifier.testTag("backToOverviewButton"),
+                  onClick = { navigationActions.navigateTo(Screen.OVERVIEW) }) {
+                    Icon(imageVector = Icons.Outlined.Home, contentDescription = "Home")
+                  }
+            })
+      },
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelect = { route -> navigationActions.navigateTo(route) },
+            tabList = LIST_TRIP_LEVEL_DESTINATION,
+            selectedItem = navigationActions.currentRoute())
+      },
+      content = { pd ->
+        Text(
+            modifier = Modifier.padding(pd).testTag("emptySettingsPrompt"),
+            text =
+                "You're viewing the Settings screen for ${trip?.name}, but it's not implemented yet.")
```

Comment:
You might add a loading or error state when trip is null, even though this part of the code might change in the future. Right now, there's no clear indication to the user when the trip data is still being fetched or if there was an error in retrieving it. Adding loading or error states will provide a better user experience, as it informs users about what's happening in the background. However, since the overall structure and implementation of this screen will evolve when the features are added, it's not a critical issue yet, but definitely a good practice to start implementing. 

Also, there's no clear indication in the code that the screen is not fully implemented. Adding a TODO comment will help maintain clarity about missing features.

Reply:
I added a Text that is displayed when selectedTrip is null. I think this is better than "Loading" since it it's an immediate error, it's not solved with time